dev/RadioButtons/RadioButtons.cpp

```diff
@@ -196,6 +198,27 @@ void RadioButtons::OnChildPreviewKeyDown(const winrt::IInspectable&, const winrt
     }
 }
 
+void RadioButtons::OnAccessKeyInvoked(const winrt::UIElement&, const winrt::AccessKeyInvokedEventArgs& args)
+{
+    if (m_selectedIndex)
+    {
+        if (auto const repeater = m_repeater.get())
+        {
+            if (auto const selectedItem = repeater.TryGetElement(m_selectedIndex))
+            {
+                if (auto const selectedItemAsControl = selectedItem.try_as<winrt::Control>())
+                {
+                    selectedItemAsControl.Focus(winrt::FocusState::Keyboard);
+                    return;
+                }
+            }
+        }
+    }
+    // If we don't have a selected index, focus the RadioButton's which under normal
+    // circumstances will put focus on the first radio button.
+    this->Focus(winrt::FocusState::Keyboard);
```

Comment:
If you successfully set Focus to something, you probably want to set args.Handled to true.  So, you probably want to set args.Handled = this->Focus(FocusState::Keyboard).  Same for the call to selectedItemAsControl above.

Also note that you can focus things that aren't Controls, such as WebView or Hyperlink.  In case you care to allow those with the code above that's trying to cast to Control first.  I think those scenarios don't apply here, but wanted to double-check.

Reply:
Correct- Hyperlink does not apply here. WebView - maybe but its hard to think of a scenario so this is probably ok.